applications/nrf_desktop/src/hw_interface/motion_sensor.c

```diff
@@ -483,50 +510,56 @@ static bool event_handler(const struct event_header *eh)
 			k_sem_give(&sem);
 
 			module_set_state(MODULE_STATE_READY);
+		} else if (state.state == STATE_DISABLED_SUSPENDED) {
+			state.state = STATE_DISABLED;
```

Comment:
Shouldn't we send `MODULE_STATE_READY`?
Btw. we should never get here (since `wake_up_event` is consumed by `power_manager` when application is in error state)

Reply:
From practical point of view this is true but I have added this part for the sake of the symmetry.